src/Workspaces/Core/Portable/Simplification/SimplificationOptions.cs

```diff
@@ -45,11 +46,32 @@ public static class SimplificationOptions
         public static Option<bool> AllowSimplificationToBaseType { get; } = new Option<bool>(NonPerLanguageFeatureName, "AllowSimplificationToBaseType", true);
 
         /// <summary>
-        /// This option says if we should simplify away the this. or Me. in member access expression
+        /// This option says if we should simplify away the <see langword="this"/>. or <see langword="Me"/>. in member access expressions.
         /// </summary>
+        [Obsolete]
         public static PerLanguageOption<bool> QualifyMemberAccessWithThisOrMe { get; } = new PerLanguageOption<bool>(PerLanguageFeatureName, "QualifyMemberAccessWithThisOrMe", defaultValue: false);
```

Comment:
I'm with @dpoeschl that UI is probably OK to drop. However, this is a real API breaking change that we should take through the appropriate channels (e.g. compatibility council).